### PR TITLE
Add test cases for ucs service function test

### PR DESCRIPTION
**Pull request by pengz1**
Add more test cases for ucs-service function test:
1. Add test cases for new asynchronous API /catalog/async and /catalog/poller
2. Add some negative test cases
3. Add a stress test case.

@anhou @bbcyyb @HQebupt @mcgG

### CODE REVIEW

File: tests/function_tests/run_test.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# Copyright 2017, Dell, Inc.
```
Comment (PengTian0) [inline tests/function_tests/run_test.sh]: duplicate copyright

File: tests/function_tests/test_ucs_api.py [hunk 2]
```diff
@@ -153,6 +153,123 @@ def test_api_ucs_power(self):
...
         self.check_all_server_power_state("up")
 
+    def test_async_api_ucs_poller(self):
+        """
+        Test the /pollers/async ucs API
+        :return:
+        """
+        api_data = request("get", "/sys")
+        self.assertEqual(api_data['status'], 200,
+                         'Incorrect HTTP return code, expected 202, got:' + str(api_data['status']))
```
Comment (PengTian0) [inline tests/function_tests/test_ucs_api.py]: expected 202 or 200?

File: tests/function_tests/test_ucs_api.py [hunk 3]
```diff
@@ -153,6 +153,123 @@ def test_api_ucs_power(self):
...
+        api_data_c = request("get", "/catalog/async", query={"identifier": "/sys/chassis-1"})
+        self.assertEqual(api_data_c['status'], 400,
+                         'Incorrect HTTP return code, expected 400, got:' +
+                         str(api_data_c['status']))
+
+
+@attr(all=True, robust=True)
+class ucs_api_stress(unittest.TestCase):
+
+    def test_async_api_ucs_catalog(self):
```
Comment (PengTian0) [inline tests/function_tests/test_ucs_api.py]: Why only stress test catalog api?
How about poller api?

**Comment by PengTian0** [comment]
+1 after comments addressed.

File: tests/function_tests/test_ucs_api.py (hunk 3 above)
Comment (pengz1) [reply tests/function_tests/test_ucs_api.py]: Any API is OK here , just to make sure UCS-service can be stressed.
It is not necessary to stress all APIs, what we want is to make sure there is no session timeout and, no timeout happens when large quantity of access happened.

**Pull request merged**